packages/replay/src/types/replayFrame.ts

```diff
@@ -125,16 +134,68 @@ export type BreadcrumbFrame =
   | FocusFrame
   | SlowClickFrame
   | MutationFrame
-  | BaseReplayFrame;
+  | BaseBreadcrumbFrame;
 
-export interface SpanFrame {
+interface BaseSpanFrame {
   op: string;
   description: string;
   startTimestamp: number;
   endTimestamp: number;
-  data: AllEntryData;
+  data?: Record<string, any>;
 }
 
+interface HistoryFrame extends BaseSpanFrame {
+  data: HistoryData;
+  op: 'navigation.push';
+}
+
+interface LargestContentfulPaintFrame extends BaseSpanFrame {
+  data: LargestContentfulPaintData;
+  op: 'largest-contentful-paint';
+}
+
+interface MemoryFrame extends BaseSpanFrame {
+  data: MemoryData,
+  op: 'memory',
+}
+
+interface NavigationFrame extends BaseSpanFrame {
+  data: NavigationData
+  op: 'navigation.navigate' | 'navigation.reload' | 'navigation.back';
```

Comment:
Ah it's `back_forward`: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/type